addon/components/frost-numeric-select.js

```diff
@@ -0,0 +1,183 @@
+/**
+ * Component definition for the numeric selection field component
+ */
+import layout from '../templates/components/frost-numeric-select'
+import Component from './frost-component'
+import Ember from 'ember'
+import {PropTypes} from 'ember-prop-types'
+import computed, {readOnly} from 'ember-computed-decorators'
+const {isPresent} = Ember
+
+export default Component.extend({
+  // == Dependencies ==========================================================
+
+  // == Keyword Properties ====================================================
+  layout,
+
+  // == PropTypes =============================================================
+
+  propTypes: {    // options
+    align: PropTypes.string,
+    isHookEmbedded: PropTypes.bool,
+    receivedHook: PropTypes.string,
```

Comment:
These two are obsolete now with the base frost component supporting `hookPrefix`, `frost-text` hasn't been updated to use `hookPrefix` yet because it would be a breaking chance, but new components should certainly not use these two.